tests/components/AddAddon.test.js

```diff
@@ -29,10 +29,6 @@ describe('<AddAddon />', () => {
         expect(renderedComponent.length).equal(1);
     });
 
-    it('Should render a dropzone component', () => {
```

Comment:
Did you intentionally remove this?

Reply:
Yes, the test passes locally but fails on travis. We have tried to resolve it but have failed. As of the moment all the PR's that were raised after rebasing yesterday have been failing due to that test.